client/src/App.jsx

```diff
@@ -1,36 +1,31 @@
 import React, { useState, useEffect } from "react";
 import Home from "./components/Home";
 import "./components/App.css";
-import { Routes, Route, Link } from "react-router-dom";
+import { Routes, Route } from "react-router-dom";
 import CraftbeerMe from "./components/CraftbeerMe";
 import AleVenture from "./components/AleVenture";
 import AleYea from "./components/AleYea";
 import Page404 from "./components/Page404";
+import NavBar from "./components/NavBar"; // Importiere die Navbar-Komponente
 
 export default function App() {
   const [flavor, setFlavor] = useState('');
   const [recommendations, setRecommendations] = useState([]);
   const [error, setError] = useState('');
 
- 
   const fetchFlavorRecommendations = async (flavor) => {
     try {
       const response = await fetch(`/api/craftbeers/${flavor}`);
       console.log('this is my fetch response for fetchFlavorRecommendations:', response);
 
       const data = await response.json();
-      console.log(data)
+      console.log(data);
       return data;
     } catch (error) {
-   
       throw new Error('Failed to fetch beer recommendations. Please try again.');
     }
   };
 
-
```

Comment:
most of the stuff made on this file, was simply refactoring and getting everything into proper syntax